### ISSUE TITLE
Input: A11y Issue - Suggestion Items inside Input are not read properly by screen reader

### ISSUE THREAD

**Issue by yzy415** [SAP/ui5-webcomponents#5824]
### Bug Description
When using Jaws and VoiceOver,

Switch to VPC mode -> press arrow keys to read through the options of the Search drop-down -> An incorrect state of "selected" is announced even before selecting the option

### Expected Behavior
The selected state should be announced only after we press the ENTER key to select the option

### Steps to Reproduce
Go to https://sap.github.io/ui5-webcomponents-react/?path=/docs/inputs-input--with-suggestion-item
Launch the Jaws or VoiceOver
Focus on the input box in the "Input with customizable SuggestionItem" section and input the letter 'b'
Use the arrow down key to read through the dropdown, you will hear "Bulgaria. List 3 of 8 items Bulgaria selected". That's to say the "selected" would be spoken regardless of the fact that the user has not selected this item using the enter key.
a11y input
### Isolated Example
https://codesandbox.io/s/web-compoennt-ui5-input-0nox78?file=/src/App.js
You can get the same result as mentioned before when inputting the letter 'b'

### Priority

- [ ]  Low
- [ ]  Medium
- [ ]  High
- [ ]  Very High

### UI5 Web Components for React Information
@ui5/webcomponents version: 1.6.0
@ui5/webcomponents-react version: 0.27.0
Operating System: MacOS
Browser: Chrome

### Additional Text
I am from SAP Successfactors, and this issue is reported by SF a11y expert.
This issue is one of [the closed old one](https://github.com/SAP/ui5-webcomponents/issues/5732).  Our A11y expert has a different perspective and left a [comment](https://github.com/SAP/ui5-webcomponents/issues/5732#issuecomment-1248351659) in the closed issue.

**Comment by ilhan007** [comment]
Hello @SAP/ui5-webcomponents-topic-rl it seems this a follo-up of https://github.com/SAP/ui5-webcomponents/issues/5732 and it's about particular input from the authors: https://github.com/SAP/ui5-webcomponents/issues/5732#issuecomment-1248351659
